org.mixedrealitytoolkit.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -577,6 +577,10 @@ private void RegisterPlacementAction()
                         }
                     }
 #pragma warning restore CS0618 // ActionBasedController and XRBaseInputInteractor.forceDeprecatedInput are obsolete
+                    else //if controllerInteractor.xrController is null then it is an interactor from a controllerless controller
+                    {
+                        controllerInteractor.selectInput.inputActionReferenceValue.action.performed += StopPlacementViaPerformedAction;
```

Comment:
we need to unregister this even handler somewhere

Reply:
Done in 527eb361c60c830306345a4c129b384ee3166cc5 , resolving.